scripts/west_commands/sign.py

```diff
@@ -109,6 +109,8 @@ def do_add_parser(self, parser_adder):
                            are currently supported''')
         group.add_argument('-p', '--tool-path', default=None,
                            help='''path to the tool itself, if needed''')
+        group.add_argument('-s', '--sign-schema', required=False, default=None,
```

Comment:
This is a tool-specific argument being added to the generic parent class; that is not acceptable.